src/compiler.ts

```diff
@@ -255,7 +253,11 @@ export const makeCompiler = (
       if (fs.existsSync(compiledPath)) {
         return
       }
+
       const starTime = new Date().getTime()
+
+      const fileName = params.compile
+      const code = params.code || fs.readFileSync(fileName, 'utf-8')
```

Comment:
Don't think this line even needs to still be here because the 'code' local variable is not actually used.

Reply:
Removed the line as requested because the `code` was really not used